src/app/user/login/login.component.ts

```diff
@@ -0,0 +1,23 @@
+import { Component } from "@angular/core";
+import { Router } from "@angular/router";
+import { AuthService } from "../shared/authentication.service";
+
+@Component({
+  templateUrl: 'login.component.html',
+  styleUrls: ['login.component.css']
+})
+
+export class LoginComponent {
+  username:any
```

Comment:
remove `any` references. Always try to find a type. Otherwise TypeScript serves no purpose.